### PR TITLE
First public version? 

### PR DESCRIPTION
**Pull request by Wolffyx**
Don't forget to replace the readme 
No guide for Categories yet
No new design for Guide part

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,167 +1,649 @@
...
-			}
-		}
-		</script>
-	</body>
+	</div>
+</section>
+<!--====  	End of Guide	   ====-->
+
+<!--==================================
+=                FAQ                 =
```
Comment (Soitora) [inline index.html]: The FAQ section not linkable per-FAQ entry, I would prefer this to be fixed before I accept the PR.

File: .editorconfig [hunk 2]
```diff
@@ -5,4 +5,4 @@ indent_style = tab
 indent_size = 4
 charset = utf-8
 trim_trailing_whitespace = true
-insert_final_newline = true
\ No newline at end of file
+# insert_final_newline = true
```
Comment (arkon) [inline .editorconfig]: Why?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,2 @@
-# soitora.github.io
+# soitora.github.io // Wolffyx.github.io
```
Comment (arkon) [inline README.md]: I suspect this whole PR needs to be rebased on top of what's currently in `master.

File: .editorconfig (hunk 2 above)
Comment (Wolffyx) [reply .editorconfig]: IDK, usually I use the editor itself to do the config but when I see an editorconfig file I only change the spaces in tabs. It still beats me this file, never read how to  make one :(

File: README.md (hunk 3 above)
Comment (Wolffyx) [reply README.md]: I will delete the readme, I saw the master has info.

**Comment by arkon** [comment]
Sorry, I made changes in `master` where all the relevant website files were moved into a `src` directory. You'll need to rebase or merge again. :P

**Comment by arkon** [comment]
Going to close this PR since the tracking branch has changed quite a bit. Feel free to add more content and such to the newer version.

**Pull request closed without merging**